plugins/destination/bigquery/client/client.go

```diff
@@ -46,20 +46,23 @@ func New(ctx context.Context, logger zerolog.Logger, destSpec specs.Destination)
 	}
 	defer client.Close()
 
-	c.projectID = spec.ProjectID
-	c.datasetID = spec.DatasetID
-	c.pluginSpec = spec
-
 	return c, nil
 }
 
 func (c *Client) bqClient(ctx context.Context) (*bigquery.Client, error) {
 	opts := []option.ClientOption{option.WithRequestReason("CloudQuery BigQuery destination")}
 	if len(c.pluginSpec.ServiceAccountKeyJSON) != 0 {
+		c.logger.Info().Msg("Using service account key JSON")
```

Comment:
Can this lead to excessive logging?

Reply:
No, it'll be logged max 3 times per sync (once during initialization, once during migration, once during sync)

Reply:
I can probably remove it now though since it doesn't provide much value.